sale_payment_mgmt/security/ir.model.access.csv

```diff
@@ -0,0 +1,7 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_payment_group_sale_salesman,account.payment,payment.model_account_payment,sales_team.group_sale_salesman,1,0,1,0
+access_account_payment_group_sale_salesman_all_leads,account.payment,payment.model_account_payment,sales_team.group_sale_salesman_all_leads,1,0,1,0
```

Comment:
Why adding this line? All the stacked groups receive same permissions as lower groups, so you only need to define the lower ACL. Only if changed, then redefine it for the upper group.

Reply:
Ok thanks, applied now to user level and work fine